octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -67,9 +82,31 @@ def set_context_object(ctx: click.Context, airbyte_url: str, workspace_id: str,
     default=True,
     help="Enable or disable telemetry for product improvement.",
 )
+@click.option(
+    "--api-headers",
+    "-h",
+    help="Additional header name and header value pairs to pass to octavia cli ex. Authorization: Basic dXNlcjpwYXNzd29yZA==",
+    multiple=True,
+    type=(str, str)
+)
+@click.option(
+    "--api-headers-file",
+    "-f",
+    envvar=AIRBYTE_HEADERS_FILE_PATH_ENV_VARIABLE_NAME,
+    help=f"Yaml file with headers configuration like below \n {API_HEADERS_YAML_FILE_INSTRUCTION}",
+    type=click.Path(exists=True, readable=True)
+)
```

Comment:
Could you please make sure that if the option is not passed and if the `envvar` is not defined the command still works? This is important for backward compatibility.

Reply:
Do you have any suggestions how to do that ? I dont see any integration tests with env variables.

Reply:
Testing it once locally without a specific test is fine for me at the moment. Just want to make sure these options are purely optional and that the behavior remains correct if they are not passed

Reply:
I did the test:
- octavia init 
- list sources and get id of the poke api source and local csv destination
- create source for poke api
- create destination for local csv
- apply connections
- create connection between mentioned above source and destination
- apply the connection
- apply once again to make sure there is no diff
- run the flow to make sure the data flows from api to csv

I performed all those steps without api-header config and file to verify if the current PR change is backward compatible. All those steps worked fine.